activestorage/app/controllers/concerns/active_storage/set_headers.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module ActiveStorage::SetHeaders #:nodoc:
+  extend ActiveSupport::Concern
+
+  private
+    def set_headers(blob)
+      expires_in 1.year, public: true
```

Comment:
How’d we land on 1 year? Should this value be configurable?

Reply:
https://developers.google.com/web/tools/lighthouse/audits/cache-policy specifically "When possible, cache immutable static assets for a long time, such as a year or longer." Does 1 year work for everybody (it works for my use case) or should we add a config option for this?

Reply:
Got it. :+1: In that case, let’s switch to [`http_cache_forever`](https://api.rubyonrails.org/classes/ActionController/ConditionalGet.html#method-i-http_cache_forever) as a stronger signal that we intend for far-future expiry (and so we pick up the [`Cache-Control: immutable`](https://www.keycdn.com/blog/cache-control-immutable) directive whenever that gets added to `http_cache_forever`).